netci.groovy

```diff
@@ -2198,6 +2340,11 @@ Constants.allScenarios.each { scenario ->
                     switch (architecture) {
                         case 'x86_arm_altjit':
                         case 'x64_arm64_altjit':
+                            // In addition, do not do innerloop testing
+                            if (scenario == "innerloop") {
```

Comment:
Doesn't the `Constants.prTriggeredValidInnerLoopCombos` code already cover this (i.e., do you need this?)?